sonaranalyzer-dotnet/rspec/cs/S3346_c#.html

```diff
@@ -3,6 +3,10 @@
 <p>In non-debug mode, all <code>Debug.Assert</code> are automatically left out. So, by contract, the boolean expressions that are evaluated by those
 assertions must absolutely not contain any side effects. Otherwise, when leaving the Debug mode, the functional behavior of the application is not the
 same anymore.</p>
+<p>The rule will raise if the method name starts with any of the following <code>remove</code>, <code>delete</code>, <code>add</code>,
+<code>pop</code>, <code>update</code>, <code>retain</code>, <code>insert</code>, <code>push</code>, <code>append</code>, <code>clear</code>,
+<code>dequeue</code>, <code>enqueue</code>, <code>dispose</code>, <code>put</code>, or <code>set</code>, although <code>SetEquals</code> will be
```

Comment:
[optional] Instead of adding it at the end of the sentence, I'd add an exception paragraph, as we do usually - it would stand out more.